R/score_forecasts.R

```diff
@@ -201,8 +205,40 @@ score_forecasts <- function(
                   "sharpness", "overprediction", "underprediction")
     })
      
-        
-      
+  # one sided     
+  sq <- scoringutils::eval_forecasts(data = joint_df,
+                                     by = observation_cols,
+                                     summarise_by = c(observation_cols, "quantile"),
+                                     interval_score_arguments = list(weigh = TRUE, count_median_twice=FALSE))%>%
+    tidyr::pivot_wider(id_cols = observation_cols,
+                       names_from = c("quantile"),
+                       values_from = c("quantile_coverage", "interval_score", abs_var, "sharpness", "overprediction", "underprediction")) %>%
+    purrr::set_names(~sub(abs_var_rename, "abs_error", .x))%>%
+    dplyr::select(
+      -dplyr::ends_with("_NA")
+    ) %>% 
+    dplyr::select(
+      -dplyr::starts_with("abs_error."),
+      -dplyr::starts_with("aem_"),
+      -dplyr::starts_with("ae_point_"),
+      -dplyr::starts_with("interval_score"),
+      -dplyr::starts_with("sharpness_"),
+      -dplyr::starts_with("underprediction_"),
+      -dplyr::starts_with("overprediction_")
+    )  
```

Comment:
could we pass a subset of these things to `eval_forecasts` via the `metrics` argument?